packages/notifications/src/NotificationPortal/NotificationPortal.tsx

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+import { Dispatch, ActionCreator } from 'redux';
+import Portal, { PortalNotificationConfig } from '../Portal';
+import { removeNotification, clearNotifications } from '../redux/actions/notifications';
+
+export interface NotificationsState {
+    [key: string]: any,
+    notifications?: PortalNotificationConfig[]
+}
+
+export interface NotificationPortalProps {
+    removeNotification: ActionCreator<PortalNotificationConfig>,
+    clearNotifications: ActionCreator<Record<string, unknown>>,
+}
+
+const mapStateToProps = ({ notifications }: NotificationsState, initialProps: NotificationsState) => ({
```

Comment:
What do you think about moving this to a `useSelector`, `useDispatch` and wrapping the `Portal` into function component?

Reply:
We could, but there is a lot of redux TS fiddling that has to be done. I don't want to deal with that right now.

Reply:
Let's create an issue and implement it later on!